content/03.results-and-discussion.md

```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
 nf-core/eager follows a similar structural foundation with the original EAGER. Given Illumina short-read FASTQ and/or BAM files, and a reference FASTA file, this can be split into four main stages:
 
 1. Preprocessing
-   1. Sequencing quality control: FastQC (@url:https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
-   2. Sequencing artefact clean-up: AdapterRemoval2 (@doi:10.1186/s13104-016-1900-2)
-   3. Preprocessing statistics generation
+   a. Sequencing quality control: FastQC (@url:https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
+   b. Sequencing artefact clean-up: AdapterRemoval2 (@doi:10.1186/s13104-016-1900-2)
+   c. Preprocessing statistics generation
 2. Mapping and post- processing
-   1. Alignment against reference genome: BWA (@doi:10.1093/bioinformatics/btp324, @doi:10.1093/bioinformatics/btp698, @arXiv:1303.3997), CircularMapper (@doi:10.1186/s13059-016-0918-z)
-   2. Mapping quality filtering: Samtools (@doi:10.1093/bioinformatics/btp352)
-   3. PCR duplicate removal: DeDup (@doi:10.1186/s13059-016-0918-z), Picard MarkDuplicates(@doi:10.1101/gr.107524.110)
-   4. Mapping statistics generation: PreSeq (@doi:10.1038/nmeth.2375), Qualimap2(@doi:10.1093/bioinformatics/btv566)
+   a. Alignment against reference genome: BWA (@doi:10.1093/bioinformatics/btp324, @doi:10.1093/bioinformatics/btp698, @arXiv:1303.3997), CircularMapper (@doi:10.1186/s13059-016-0918-z)
```

Comment:
Nope, this breaks markdown, please revert.

Reply:
OOF, my bad! Converted to bullet points so there aren't nested numbers.